<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckSettings</class>
 <widget class="QWidget" name="CheckSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckSettings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="travelOverlaps">
     <property name="text">
      <string>Reiseüberschneidungen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="tooManyHotels">
     <property name="text">
      <string>Überflüssige Hotelbuchungen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="tooLittleHotels">
     <property name="text">
      <string>Fehlende Hotelbuchungen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="tooManyRentalCars">
     <property name="text">
      <string>Überflüssige Mietwagenbuchungen</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
